### PR TITLE
Adding information about Audit Logs

### PR DESCRIPTION
**Pull request by kaliserichmond**
## Summary
Adding information to rbac/overview.md doc about Enterprise only feature Audit Logs.

## Changes
Adding information to rbac/overview.md doc.

## Importance
Adding documentation.

## Checklist

This PR:

- [ ] adds new tests (if appropriate)
- [ ] adds a change file in the `changes/` directory (if appropriate)
- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

File: docs/orchestration/rbac/overview.md [hunk 1]
```diff
@@ -127,3 +127,26 @@ query {
 ```
 
 The `memberships.id` value for a given user should be provided as the `membership_id` parameter when calling `set_membership_role`.
+
+### Audit Log (Enterprise Only)
+
+Separately from task run or flow run logs, Prefect has an Enterprise only feature to track “audit logs” for events within the system per tenant.
```
Comment (tpdorsey) [inline docs/orchestration/rbac/overview.md:133]:
```suggestion
Separately from task run or flow run logs, Prefect has an Enterprise-only feature to track “audit logs” for events within the system, per tenant. 
```

File: docs/orchestration/rbac/overview.md [hunk 2]
```diff
@@ -127,3 +127,26 @@ query {
 ```
 
 The `memberships.id` value for a given user should be provided as the `membership_id` parameter when calling `set_membership_role`.
+
+### Audit Log (Enterprise Only)
+
+Separately from task run or flow run logs, Prefect has an Enterprise only feature to track “audit logs” for events within the system per tenant. 
+
+Audit logs for each tenant have a `timestamp`, `message` describing the event, an `object_id` associated with an `object_table` and an option `info`.
```
Comment (tpdorsey) [inline docs/orchestration/rbac/overview.md:135]:
```suggestion
Audit logs for each tenant have a `timestamp`, a `message` describing the event, an `object_id` associated with an `object_table`, and an option `info`.
```

Comment (tpdorsey) [reply docs/orchestration/rbac/overview.md:135]: is it "optional 'info'"?

File: docs/orchestration/rbac/overview.md [hunk 3]
```diff
@@ -127,3 +127,26 @@ query {
...
+
+An example audit log for a user logging in would contain a timestamp, the login event message, the object table would be 'user' and the object id would be user id of the user logging in.
```
Comment (tpdorsey) [inline docs/orchestration/rbac/overview.md:137]:
```suggestion
An example audit log for a user logging in would contain a timestamp, the login event message, the object table would be 'user', and the object id would be user id of the user logging in.
```

File: docs/orchestration/rbac/overview.md [hunk 4]
```diff
@@ -127,3 +127,26 @@ query {
...
+
+
+Audit logs can accessed via the GraphQL API.
```
Comment (tpdorsey) [inline docs/orchestration/rbac/overview.md:140]:
```suggestion
Audit logs can be accessed via the GraphQL API.
```

**Review by tpdorsey — COMMENTED** [review]
Looks good. A few minor suggested edits.

File: docs/orchestration/rbac/overview.md (hunk 2 above)
Comment (zangell44) [reply docs/orchestration/rbac/overview.md:135]: I would say something like "an optional `info` field, which contains information about who performed an action (e.g. the user who turned a flow's schedule on)."

File: docs/orchestration/rbac/overview.md [hunk 5]
```diff
@@ -127,3 +127,26 @@ query {
 ```
 
 The `memberships.id` value for a given user should be provided as the `membership_id` parameter when calling `set_membership_role`.
+
+### Audit Log (Enterprise Only)
+
+Separately from task run or flow run logs, Prefect has an Enterprise-only feature to track “audit logs” for events within the system, per tenant. 
+
+Audit logs for each tenant have a `timestamp`, a `message` describing the event, an `object_id` associated with an `object_table`, and an option `info`.
```
Comment (tpdorsey) [inline docs/orchestration/rbac/overview.md:135]:
```suggestion
Audit logs for each tenant have a `timestamp`, a `message` describing the event, an `object_id` associated with an `object_table`, and an optional `info` field.
```

File: docs/orchestration/rbac/overview.md [hunk 6]
```diff
@@ -127,3 +127,26 @@ query {
...
+
+An example audit log for a user logging in would contain a timestamp, the login event message, the object table would be 'user', and the object id would be user id of the user logging in.
```
Comment (tpdorsey) [inline docs/orchestration/rbac/overview.md:137]:
```suggestion
An example audit log for a user logging in would contain a timestamp, the login event message, the object table would be 'user', and the object id would be the user id of the user logging in.
```

**Review by tpdorsey — APPROVED** [review]
Looks good. Two minor suggested changes.

**Pull request merged**